examples/trivial/app/BUILD.bazel

```diff
@@ -7,7 +7,10 @@ kt_jvm_library(
         "//:java_deps",
     ],
 )
-
+kt_kotlinc_options(
```

Comment:
in #422 you proposed 
```
kt_compiler_options(
   opt_in = [
        "boo.bar.Baz"
   ]
)
```
but there is no kt_compiler_options construct currently. It felt strange to introduce that rather than just use kotlinc_options so I added it there

Reply:
🤦  I meant the kotlinc options. The naming is new.